src/components/CardContainer/CardContainer.js

```diff
@@ -30,12 +30,28 @@ class CardContainer extends React.Component {
             .then(res => {
                 let random = UniqueRandomArray(res)
```

Comment:
(I know this is leftover from a previous story but) This is quite a vague name, can we call it something more semantic?

Reply:
on it

Reply:
I think this is actually a function, so maybe `getRandomDog` or something is maybe more semantic?

Reply:
on it